README.md

```diff
@@ -21,20 +24,15 @@ Unit testing of the application is performed within
 [yarn](https://yarnpkg.com/) is used to manage the build/test/deploy cycle.
 
 #### Screenshot
-
-![screen August 17, 2017](documentation/screenshot.png)
+![Consumer Financial Protection Bureau - Google Chrome 20-10-2023 10_37_09 PM](https://github.com/kiranrokkam09/ccdb5-ui/assets/85286397/90b9e381-f960-4c49-b956-0b8a254bcae3)
```

Comment:
The screenshot image that should be replaced is https://github.com/cfpb/ccdb5-ui/blob/main/documentation/screenshot.png

The linked screenshot is a screenshot of the homepage, not this app.

I'd say revert this for now if you're not able to update the screenshot from a fork.